frontends/ol-components/src/components/Button/Button.tsx

```diff
@@ -133,6 +139,17 @@ const ButtonStyled = styled.button<ButtonStyleProps>((props) => {
         color: colors.silverGray,
       },
     },
+    variant === "noBorder" && {
+      backgroundColor: colors.white,
+      color: colors.darkGray2,
+      border: "none",
+      ":hover:not(:disabled)": {
+        backgroundColor: tinycolor(colors.darkGray1).setAlpha(0.06).toString(),
```

Comment:
I didn't see a hover state specified in figma. This looks fine, but I did leave a note for steve/bilal jkachel/4599-mobile-filter-ctas